### ISSUE TITLE
Parsing error

### ISSUE THREAD

**Issue by magicmakerman** [Devilx86/ImpostorModmenu#11]
I tried downloading the 1.3 Among Us Mod but I keep getting parsing error. Is there a way to fix this? I also tried 1.2 and it wouldn't install at all. It kept saying App Not Installed. I have used 3 other mods for this game and this is the only one that I can't get to work. I think this one is far superior to the others. Please help me with this. I also tried just the 1.2 from this site and it would open only after I signed the app, but it only ran for about 2 seconds and force closed. I can't get anything to work from you.

**Comment by Redlything** [comment]
Umm 1.3 update isn't out yet.Its not visible on the page

**Comment by Devilx86** [comment]
@Redlything it is but not on git because I am trying to fix the parse issue that a lot of people have contacted me about in android 11, will push it once I finish fixing that.

**Comment by Milionerzy11** [comment]
> I tried downloading the 1.3 Among Us Mod but I keep getting parsing error. Is there a way to fix this? I also tried 1.2 and it wouldn't install at all. It kept saying App Not Installed. I have used 3 other mods for this game and this is the only one that I can't get to work. I think this one is far superior to the others. Please help me with this. I also tried just the 1.2 from this site and it would open only after I signed the app, but it only ran for about 2 seconds and force closed. I can't get anything to work from you.

@magicmakerman

**Comment by Devilx86** [comment]
Hello, I finally fixed this, 
TLDR:
`zip align apk and use apksigner instead of jarsigner`

In the release version that I posted for v1.3, I had some patches in the Manifest to fix Android 11 issue as I was told by someone that this patched worked for them in v1.2. However this was not why v1.2/v1.3 did not work on Android 11, hence these patches were useless. Unfortunately I did not have android 11 with me to test it out/fix it. After I was finally able to set up an android 11 emulator. I noticed `adb install amongus-impostor-v1.3.apk` gave me an error related to `resource.arsc` not being 4 byte aligned. Using `zipaligner -c amongus-impostor-v1.3.apk` to verify, I noticed the `resources.arsc` section was `BAD-1` so I zip aligned the APK. However after signing with `jarsigner` in the `inject-apk.sh` script the section turned `BAD` again and `zip aligning` once again would result in some of the package contents to be unsigned. So I solved this by zip aligning and signing with `apksigner` instead of `jarsigner`

I will update the script and the code for v1.3 here tommorrow
